meson.build

```diff
@@ -272,8 +272,14 @@ int main() {
  return 0;
 }
 '''
+if meson.is_cross_build()
+  have_ieee754 = cc.get_id() in ['gcc', 'clang']
```

Comment:
I think you want [`meson.get_cross_property()`](https://mesonbuild.com/Reference-manual_builtin_meson.html#mesonget_cross_property) here:

```
ieee754_float_div = meson.get_cross_property('ieee754_float_div', cc.get_id() in ['gcc', 'clang'])
```

and use `ieee754_float_div` instead of `have_ieee754`.

Reply:
good idea

Reply:
I wonder if it fits there, as you may break it with extra cflags I guess

Reply:
If you pass additional compiler flags without checking the implications then there's not much we can do about it